API/Controllers/AddInDBController.cs

```diff
@@ -211,18 +212,27 @@ public async Task<IActionResult> CreateExamRequest([FromBody] CreateExamRequestD
                 Duration = examRequestDto.Duration,
                 Details = examRequestDto.Details,
                 Status = examRequestDto.Status,
-                CreationDate = DateTime.UtcNow,
-                ExamRequestRooms = rooms.Select(r => new ExamRequestRoom
-                {
-                    RoomID = r.RoomID
-                }).ToList()
+                CreationDate = DateTime.UtcNow
             };
             examRequest.Session = session;
             examRequest.Course = course;
             examRequest.Group = group;
             examRequest.Assistant = labHolder.Professor;
 
             _context.ExamRequests.Add(examRequest);
+
+            // Insert Exam Requested Rooms into ExamRequestRooms table
+            foreach (var room in rooms)
+            {
+                var examRequestRooms = new ExamRequestRooms
+                {
+                    ExamRequestID = examRequest.ExamRequestID,
+                    RoomID = room.RoomID
+                };
+                _context.ExamRequestRooms.Add(examRequestRooms);
+            }
+
+            // Commit changes to the database
```

Comment:
Mai sus era ok

Reply:
Desi plasez datele in acelasi *_context*, le introduc totusi in tabele diferite:
- ExamRequests
- ExamRequestRooms